src/api/json/catalog.json

```diff
@@ -2473,6 +2473,17 @@
       ],
       "url": "https://gitlab.com/gitlab-org/gitlab/-/raw/master/app/assets/javascripts/editor/schema/ci.json"
     },
+    {
+      "name": "Gitpod Automations",
+      "description": "Configuration for Gitpod Automations",
+      "fileMatch": [
+        "automations.json",
```

Comment:
These are pretty generic file names. Can you make them more specific by adding a folder name to the pattern, for example?

Reply:
Thanks, I've now updated the PR with a `.gitpod` dir prefix. Let me know if that works.

Reply:
@madskristensen This has been fixed, could you PTAL?